features/restaurant_list.feature

```diff
@@ -10,9 +10,10 @@ Feature: User can see a list of restaurants on the landing page
       |Oliver's Burger  |
       |Borat's Palace   |
 
-
   Scenario:
     When I visit the landing page
     Then I would like to see "Thomas Kebabrulle"
     And I would like to see "Oliver's Burger"
     And I would like to see "Borat's Palace"
+    When I click on "Show page" by "Thomas Kebabrulle"
```

Comment:
Why are you adding these steps to this scenario? Isn't this feature about listing restaurants on the landing page only? The 2 steps you're adding here are already tested in the `restaurant_page.feature` so no need to do it again here

Reply:
We have an overlap from landing page so guessing we need to cherry pick commits?